classes/article/SubmissionFileDAO.inc.php

```diff
@@ -64,6 +64,8 @@ function getGenreCategoryMapping() {
 	function fromRow($row) {
 		if (isset($row['artwork_file_id']) && is_numeric($row['artwork_file_id'])) {
 			return parent::fromRow($row, 'SubmissionArtworkFile');
+		} elseif (isset($row['supplementary_file_id']) && is_numeric($row['supplementary_file_id'])) {
```

Comment:
Bruno, thinking about this a little more: OJS doesn't provide supplementary files as their own OAI records; each OAI record is a single article. The OAI data refers to supplementary files as "relation" elements but that's all. I think that's the appropriate thing to do for articles, until we have a requirement for more. However, the DC metadata will be active in other areas for supplementary content, such as Dataverse.